### DISCUSSION TITLE
helloDesktop on Haiku?

### DISCUSSION THREAD

**Discussion by probonopd** [helloSystem/hello#233] [Ideas]
https://medium.com/@probonopd/my-first-day-with-haiku-shockingly-good-8930cad4bbb0

Qt has been ported there.

But the community is more interested in a BeOS than a Mac like user experience.

And there is no graphics acceleration.

**Comment by kettle-7** [comment]
And the desktop environment is part of the kernel. Nonetheless there is probably some stuff we could take away from it, like a fuse filesystem mimicking pkgfs.
